sencha-workspace/SlateTasksStudent/app/view/TaskTree.js

```diff
@@ -171,39 +195,30 @@ Ext.define('SlateTasksStudent.view.TaskTree', {
         }
     },
 
+
     // config handlers
     updateCourseSection: function(val) {
         var me = this;
 
         me.fireEvent('coursesectionchange', me, val);
     },
 
-    applyData: function(data) {
-        data.statusClasses = this.getStatusClasses();
-        return data;
-    },
-
-    updateData: function(data) {
-        this.update(data);
-    },
 
     // event handlers
     onTreeClick: function(ev, t) {
         var me = this,
             target = Ext.get(t),
-            parentEl, recordId;
+            parentEl = target.up('.slate-tasktree-item'),
```

Comment:
@BillClinton use `ev.getTarget('.slate-tasktree-item', me.el, true)` to get an Ext.Element reference to the element containing the event matching a given selector. You don't need the `Ext.get(t)` reference declared above